packages/sdk/python/human-protocol-sdk/README.md

```diff
@@ -1,150 +1,233 @@
 # Human Protocol Python SDK
 
-Python SDK to launch/manage jobs on [Human Protocol](https://www.humanprotocol.org/)
+Python SDK to interact with [Human Protocol](https://www.humanprotocol.org/)
 
 ## Installation
 
-### Manual
+[Python3](https://www.python.org/) is required.
 
-You need few essential system requirements to successfully install our Python 3 package.
+```bash
+pip install human-protocol-sdk
+```
 
-### Annoying testing feature
+## Getting Started
 
-Rightly or Wrongly we tried to use doctests for the vast majority of testing in this project
-. As a result you may have to remove the raise_on_error=True in the module you are testing
-to get good feedback on what's broken.
+### Staking
 
-#### Debian / Ubuntu
+Before creating an escrow, the you need to stake HMT tokens to become a valid entity on Human Protocol.
```

Comment:
"the you need to stake HMT tokens to become a valid entity on Human Protocol" --> "you need to stake HMT tokens to become a valid entity on Human Protocol"